iOS/DittoChat/Data/DittoService.swift

```diff
@@ -531,33 +531,34 @@ extension DittoService {
     
     func joinPrivateRoom(qrCode: String) {
         let parts = qrCode.split(separator: "\n")
-        guard parts.count == 3 else {
-            print("DittoService.\(#function): Error - expected 3 parts to QR code: \(qrCode) --> RETURN")
+        guard parts.count == 7 else {
+            print("DittoService.\(#function): Error - expected 7 parts to QR code: \(qrCode) --> RETURN")
             return
         }
         // parse qrCode for roomId, collectionId, messagesId
         let roomId = String(parts[0])
         let collectionId = String(parts[1])
         let messagesId = String(parts[2])
+        let roomName = String(parts[3])
+        let isPrivate = Bool(String(parts[4])) ?? true
+        let createdBy = String(parts[5])
+        let createdOnString = String(parts[6])
+        
+        let dateFormatter = DateFormatter()
```

Comment:
Please use the extension instance for consistency:
`DateFormatter.isoDate.string(from: createdOnString)`